src/elements/hx-switch-control/index.spec.js

```diff
@@ -0,0 +1,75 @@
+import { fixture, oneEvent, expect } from '@open-wc/testing';
+
+/**
+ * <hx-switch-control> component tests
+ *
+ * @type HXSwitchControlElement
+ *
+ */
+describe('<hx-switch-control> component tests', () => {
+    const template = '<hx-switch-control>';
+    const mockup = `<hx-switch-control>
+                        <input type="checkbox" id="switchOnOffTest" />
+                        <label for="switchOnOffTest">
+                        <hx-switch
+                            onlabel="on"
+                            offlabel="off"
+                            aria-labelledby="switchOnOffTest">
+                        </hx-switch>
+                        </label>
+                    </hx-switch-control>`;
+
+    describe('test instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const fragment = /** @type {HXSwitchControlElement} */ await fixture(mockup);
+            const attr = fragment.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const fragment = /** @type {HXSwitchControlElement} */ await fixture(mockup);
+            const prop = fragment.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered Light DOM should NOT equal simple template ${template}`, async () => {
+            const fragment = /** @type {HXSwitchControlElement} */ await fixture(mockup);
+
+            expect(fragment).lightDom.to.not.equal(mockup);
+        });
+
+        it(`should NOT have a Shadow DOM`, async () => {
+            const fragment = /** @type {HXSwitchControlElement} */ await fixture(mockup);
+            const shadow = fragment.shadowRoot;
+
+            expect(shadow).to.be.null;
+        });
+
+    });
+    describe('test for click event listener', () => {
+        it('should fire a click event', async () => {
+            const fragment = /** @type { HXSwitchControlElement } */ await fixture(mockup);
+            const detail = { evt: 'clicked!'};
+            const customEvent = new CustomEvent('click', { detail });
+
+            setTimeout(() => fragment.dispatchEvent(customEvent));
+            const evt = await oneEvent(fragment, 'click');
+
+            expect(evt).to.equal(customEvent);
+            expect(evt.detail).to.equal(detail);
+        });
+    });
+
+    describe('test Control Element', () => {
```

Comment:
Please add test for:

https://github.com/HelixDesignSystem/helix-ui/blob/6ac257bf730b79eb7b55da76800a7e39def34f85/src/elements/hx-switch-control/index.js#L41-L58

* get toggled
* get switchElement

Reply:
Changes have been done, please review.